lua/nvim-treesitter/parsers.lua

```diff
@@ -2624,6 +2624,22 @@ list.templ = {
   maintainers = { "@vrischmann" },
 }
 
+list.runescript = {
+  install_info = {
+    url = "https://github.com/2004Scape/tree-sitter-runescript",
+    files = { "src/parser.c", "src/scanner.c" },
+  },
+  maintainers = { "@2004Scape" },
+}
+
+list.clientscript = {
+  install_info = {
+    url = "https://github.com/2004Scape/tree-sitter-runescript",
```

Comment:
The two languages indeed inherit the same syntax, but are counted as two separate languages (legacy reasons that only the devs making the runescape engine back in 2004 know any answers to).

Reply:
If they're truly identical, then you only need to add it once and handle this on the `vim.treesitter.language.register` level (see the table at the top of the file).

Reply:
(In case that wasn't clear from the test failures: you _cannot_ do that (add the same parser twice under different names); the language name here **must** be the same as the name the grammar itself specifies. Please drop the "clientscript" entry and (if appropriate) add an entry in the alias table.

Reply:
Yes, it was clear. I will submit a revision when I have time (by eod).

Reply:
Ok, didn't want to push, take your time -- just explain a bit more.